include/conf/find.hpp

```diff
@@ -19,33 +30,19 @@ namespace color_coded
       fs::directory_iterator const end;
 
       static auto constexpr file_name(".color_coded");
+      static auto constexpr compilation_database("compile_commands.json");
       auto const typed_file_name(std::string{ file_name } + "_" + filetype);
+      std::string const flag_files[] = {file_name, compilation_database, typed_file_name};
```

Comment:
Please `s/ = //` here, to be consistent. color_coded always uses direct initialization.